www/docs/src/components/sidebar/app-sidebar-sublist.tsx

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable */
```

Comment:
Let's remove this and use `eslint-disable-next-line` instead (or do some typescript casting to keep the compiler happy).